.pipelines/pipeline.yaml

```diff
@@ -330,25 +336,45 @@ stages:
       clusterName: "swifte2e"
       osSku: "Ubuntu"
 
-  - template: singletenancy/aks-engine/e2e-job-template.yaml
+  - template: singletenancy/aks/e2e-job-template.yaml
     parameters:
       name: "ubuntu_18_04_linux_e2e"
-      displayName: Ubuntu 18.04
-      pipelineBuildImage: "$(BUILD_IMAGE)"
-      clusterDefinition: "cniLinux1804.json"
-      clusterDefinitionCniTypeKey: "azureCNIURLLinux"
-      clusterDefinitionCniBuildOS: "linux"
-      clusterDefinitionCniBuildExt: ".tgz"
+      displayName: AKS Ubuntu 18.04
+      arch: amd64
+      os: linux
+      clusterType: linux-cniv1-up
+      clusterName: "ubuntu18e2e"
+      k8sVersion: 1.24.9
 
-  - template: singletenancy/aks-engine/e2e-job-template.yaml
+  - template: singletenancy/aks/e2e-job-template.yaml
     parameters:
       name: "windows_19_03_e2e"
-      displayName: "Windows 1903"
-      pipelineBuildImage: "$(BUILD_IMAGE)"
-      clusterDefinition: "cniWindows1903.json"
-      clusterDefinitionCniTypeKey: "azureCNIURLWindows"
-      clusterDefinitionCniBuildOS: "windows"
-      clusterDefinitionCniBuildExt: ".zip"
+      displayName: AKS Windows 1903
+      arch: amd64
+      os: linux
+      clusterType: windows-cniv1-up
+      clusterName: "win19e2e"
+      windowsOsSku: Windows2019
```

Comment:
Nit: Is this testing windows 1903 or 2019

Reply:
Windows 2019 update version was 1903.